pkg/commands/refresh.go

```diff
@@ -152,6 +194,13 @@ func refreshResultReport(newCaps, oldCaps []types.Capability) map[refreshStatus]
 		for _, newCap := range newCaps {
 			if oldCap.Name == newCap.Name {
 				found = true
+				if h, ok := cachedHash[newCap.Name]; ok {
+					// use cached hash to determine whether the cap is changed
+					if h == newHash[newCap.Name] {
+						report[unchanged] = append(report[unchanged], newCap)
+						break
+					}
+				}
 				if types.EqualCapability(oldCap, newCap) {
```

Comment:
do we still need this when we have already use cachedHash ?

Reply:
Yes, I think we can remove it. It seems unlikely to delete the cache file by someone.

Reply:
After reconsideration, I think it's okay to keep this part in case of cached hash is missing.